### DISCUSSION TITLE
Unable to create Json type when creating jdbc external table

### DISCUSSION THREAD

**Discussion by Carey8175** [apache/doris#29973] [A - General / Q&A]
求助大神们，我在建立external table时，尝试指定一个字段为json类型，但是Doris会报错，告诉我这是key列。难道对于外表来讲都是key列吗？我应该怎么做。

`CREATE EXTERNAL TABLE `customer_test_json`(
`token` text NOT NULL,
 `work_wx_group_name` json NULL,
)
ENGINE=JDBC
PROPERTIES(
	"resource" = "resource_postgresql",
	"database" = "public",
	"table" = "customer",
	"table_type" = "postgresql"
)
> 1105 - errCode = 2, detailMessage = JSONB type should not be used in key column[work_wx_group_name].
> Time: 0.028s

**Comment by LemonLiTree** [comment]
use catalog.
